analysis_options.yaml

```diff
@@ -1,7 +0,0 @@
-analyzer:
-  strong-mode:
-    implicit-casts: false
-    implicit-dynamic: false
-linter:
-  rules:
-    - slash_for_doc_comments
```

Comment:
we should copy our "standard" analysis options here

Reply:
Done